core/controllers/general_feedback.py

```
@@ -288,6 +282,223 @@ def post(self) -> None:
+                'Feedback with ID %s does not exist.' % report_id
+            )
+        try:
+            general_feedback_services.validate_platform_feedback_belongs_to_dashboard(
+                feedback=feedback,
+                dashboard=dashboard,
+                dashboard_id=dashboard_id,
+            )
+        except ValueError:
+            raise self.NotFoundException(
+                'Feedback with ID %s does not exist.' % report_id
+            )
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)